examples/sks-with-private-network/main.tf

```
@@ -0,0 +1,28 @@
+# Customizable parameters
+locals {
+  my_zone = "ch-gva-2"
+}
+
+resource "exoscale_private_network" "my_private_network" {
+  zone = local.my_zone
+  name = "test-private-network"
```

[nitpick] Naming consistency:
```suggestion
  name = "my-private-network"
```